services/telemetry/telemetry.go

```diff
@@ -1283,6 +1283,7 @@ func (ts *TelemetryService) trackPluginConfig(cfg *model.Config, marketplaceURL
 		"com.mattermost.custom-attributes",
 		"com.mattermost.mscalendar",
 		"com.mattermost.nps",
+		"com.mattermost.plugin-channel-export",
```

Comment:
Do we need this, even if there is no telemetry in the plugin?

Reply:
I /think/ we whitelist the plugin here so that it gets included in the metrics as being installed.

Reply:
Correct. Please notes that this list is only used for airgapped installations.